packages/now-client/tests/fixtures/vercelignore-allow-nodemodules/.vercelignore

```diff
@@ -1,6 +1,6 @@
 # literally dont ignore the node_modules directory
 # so basically include the node_modules directory recursively
-!node_modules/
+!node_modules
```

Comment:
This could be considered a breaking change, so we may want to wait for a major version bump before merging this.